src/main/java/frc/robot/commands/ClimbUpCommand.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+import frc.robot.subsystems.Climb;
+
+public class ClimbUpCommand extends Command {
+    private Climb climb = Climb.getInstance();
+
+    public ClimbUpCommand() {
+        super.addRequirements(climb);
+    }
+
+    @Override
+    public void execute() {
+        climb.runClimbUp();
+    }
+
+    @Override
+    public boolean isFinished() {
+        return climb.getSwitch();
```

Comment:
```suggestion
        return false;
```
The switch is at the bottom of travel, not the top